content/tutorial/01-svelte/04-logic/05-keyed-each-blocks/README.md

```diff
@@ -2,9 +2,9 @@
 title: Keyed each blocks
 ---
 
-By default, when you modify the value of an `each` block, it will add and remove items at the _end_ of the block, and update any values that have changed. That might not be what you want.
+By default, when you modify the value of an `each` block, such as adding or removing items, it will apply such modifications at the _end_ of the block, and then update any values that have changed. That might not be what you want.
```

Comment:
```suggestion
By default, when you modify the value of an `each` block, it will add and remove items at the _end_ of the block, and update any values that have changed. That might not be what you want.
```

I think this one reads better as is